eeps/eep-0000.md

```diff
@@ -62,7 +62,7 @@ Index
 Copyright
 =========
 
-This document has been placed in the public domain.
+This document is placed in the public domain or under the CC0-1.0-Universal license, whichever is more permissive.
```

Comment:
Line break this line to keep lines under 79 chars, but try to fill up to 70

Reply:
Have added new commit where the line is split